packages/gdk_pixbuf.rb

```diff
@@ -50,19 +37,19 @@ def self.install
     loader_dir = "#{CREW_DEST_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders"
     FileUtils.mkdir_p loader_dir
     system "touch #{loader_dir}.cache"
+
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
+    @env = <<~EOF
+      # gdk-pixbuf configuration
+      export GDK_PIXBUF_MODULEDIR=#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders
+      export GDK_PIXBUF_MODULE_FILE=#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders.cache
+    EOF
+    IO.write("#{CREW_DEST_PREFIX}/etc/env.d/gdk_pixbuf", @env)
   end
 
   def self.postinstall
     ENV['GDK_PIXBUF_MODULEDIR'] = "#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders"
     ENV['GDK_PIXBUF_MODULE_FILE'] = "#{ENV['GDK_PIXBUF_MODULEDIR']}.cache"
     system 'gdk-pixbuf-query-loaders --update-cache'
-    gdk_pixbuf_in_bashrc = `grep -c "GDK_PIXBUF_MODULEDIR" ~/.bashrc || true`
-    unless gdk_pixbuf_in_bashrc.to_i.positive?
-      puts 'Putting GDK_PIXBUF code in ~/.bashrc'.lightblue
-      system "echo 'export GDK_PIXBUF_MODULEDIR=#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders' >> ~/.bashrc"
-      system "echo 'export GDK_PIXBUF_MODULE_FILE=#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders.cache' >> ~/.bashrc"
-      puts 'To complete the installation, execute the following:'.orange
-      puts 'source ~/.bashrc'.orange
-    end
```

Comment:
Bump gdk-pixbuf version?

Reply:
New file stored in the package metadata?

Reply:
I updated this file to the current version with binaries.